### PR TITLE
Fixed #4

### LINKED ISSUE #4: Sort Specification and Configuration dropdown entries lexicographically
Currently, the entries in the dropdowns for specification and configuration of CPAchecker are not sorted.

E.g., in the specification dropdown, the order is: "MainEntry, TerminatingFunctions, deadlock",
and in the configuration dropdown, the order is: "witness-validation, witnessValidation, apronAnalysis-Cegar" .

For easier search, these should be sorted lexicographically.

In addition, since configurations often have one short version with a 'best by experience' configuration and multiple alternatives (e.g., `valueAnalysis` and `valueAnalysis-featureVars`),
it would be nice to have the short version shown first (but this should actually be the default for lexicographic sort).

### PR DESCRIPTION
**Pull request by SupunArunoda**

### CODE REVIEW

File: org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public static void cleanTempDirectory() {
 			@Override
 			public FileVisitResult visitFile(Path path, BasicFileAttributes attrs) throws IOException {
 				if (pathMatcher.matches(path)) {
-					String tempPath = (path.toString().replaceAll(location + "/", "")).toLowerCase();
+					String tempPath = StringHandler.toCamelCase((path.toString().replaceAll(location + "/", "")).toLowerCase());
```
Comment (leostrakosch) [inline org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java]: Why is the camel case necessary? Why don't you take the file name, as is?

Comment (leostrakosch) [reply org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java]: (without the `toLowerCase()`, of course)

Comment (leostrakosch) [reply org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java]: So if  you only take `path.toString().replaceAll(location + "/", "")`, why isn't it working? What is the observed behavior?

**Comment by leostrakosch** [comment]
By the code, it looks good - did you check whether it works as described?

**Comment by leostrakosch** [comment]
Hi @SupunArunoda , did you check whether the code works as described in #4 ?
If so, you can merge the pull request.

**Comment by SupunArunoda** [comment]
Hi Thomas,
I have checked. I was waiting for your approval to merge.

Thanks

**Comment by leostrakosch** [comment]
Is it not possible for you to merge the pull request yourself?
Otherwise, I gave you my approval above, so go ahead 😉

**Pull request merged**